README.md

```diff
@@ -0,0 +1,104 @@
+# **ALTERNATE BIT PROTOCOL IMPLEMENTATION IN CADMIUM**
+---
+### FILES ORGANIZATION
+---
+### Carleton University
+#### Authors
+---
+**Main Developer**: Cristina Ruiz Martin
+**Modified by**: Tarim Islam & Navid Hossain
+### Introduction
+---
+### Alternate Bit Protocol (ABP) is a protocol which is connectionless for transfering pair of protocol entities. It is a simple form of sliding window protocol.Interation of two pairs in a message is based on sequence of 0 & 1. 
+#### **Receiver**- It receives the data and sends back acknowledgement after certain perios of time.
+#### **Subnets**- It just transmits the packets to next node after a time delay.
+#### **Sender**-  It sends the packet and then waits for acknowledgement from the receiver but if it does not receive the acknowledgement within time window it will resend the packet with alternating bit attached to it so that it can differentiate between the two packets. Once it receives the acknowledgement it will send the next packet until it has no more packets to send.
+---
+### File Management
+1. src [This folder contains the source code for the message and top model]
+2. data_structures [Contains the source code for the top message]
+3. top_model [Contains the source code for the top_model]
+4. lib [This folder contains the third party header files]
+5. vendor[This folder contains 3rd party header files needed in the project]
+   5.1 iestream.hpp
+   5.2 NDTime.hpp
+   5.3 cadmium-master[contains cadmium files]
+   5.4 DESTimes-master[contains destimes files]
+	
+### include [This folder contains the header files used]
+	1. atomics
+		1.1 receiverCadmium.hpp
+		1.2 senderCadmium.hpp
+		1.3 subnetCadmium.hpp
+	2. data_structures [contains message header file]
+		2.1 message.hpp
+		
+### test [This folder the unit test for the different include files]
+1. data [This folder contains test input output files]
+		
+		1.1 receiver [This folder contains the input output files foe the unit test of the receiver]
+		1.2 sender [This folder contains the input output files foe the unit test of the sender]
+		1.3 subnet [This folder contains the input output files foe the unit test of the subnet]
+		
+2. src [This folder contains the source code for the sender,receiver and subnet]
+		
+		2.1 receiver[This folder contains the source code for the receiver]
+			main.cpp
+			
+		2.2 sender[This folder contains the source code for the sender]
+			main.cpp
+			
+		2.3 subnet[This folder contains the source code for the subnet]
+			main.cpp
+			
+3. doc [contains documents related to the project]
+
+#### data [contains input/output files required/generated by the ABP simulator]
+	1. input_abp_0.txt
+	2. input_abp_1.txt
+	3. abp_output_0.txt
+	4. abp_output_1.txt
+### makefile [make file to compile the project]
+	
+### Steps to Run Simulations
+---
+**alternatebitprotocol.doc contains the explanation of this simulator**
+       
+       1.   Run the unit test
```

Comment:
Your "Run the unit test" looks like it is in the code block. Also, kindly check its indentation

Reply:
WE did the change correct the indentation